src/cmd-koji-upload

```diff
@@ -598,13 +598,19 @@ class Upload(_KojiBase):
 
         now = datetime.datetime.utcnow()
         stamp = now.strftime("%s")
-        self.release = now.strftime("%H%M%S")
+        # We use release as '0' so we can easily guess the NVR name when we need to.
+        # The Brew NVR is generated by:
+        #   -  {package_name}-{rhcos_version}-{release}
+        #   -  rhcos-x86_64-412.86.202302162021.0-0
+        # Seems there is no option for this type of Build to add arch as
+        # part of the NVR.
+        # That's the reason we use 'rhcos-arch' as package name (this was also a
+        # request from ProdSec).
+        self.release = "0"
```

Comment:
Maybe let's consider setting this as a global variable at the top of this file and then we can comment it once and use it throughout.

The reason I ask this is because I feel like setting `release = "0"` on line 432 is also confusing without the comment, but if we set the value once at the top with the comment there then it's easy to correlate.